packages/e2e/local-network/templates/blockfrost-ryo/byron_genesis.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
More or less as some comments ago... I would have expected blockfrost-ryo to be an additional service inside current local-network.

Is there some issue preventing blockfrost-ryo to work in current local-network? (obviously apart from blockfrost-ryo is not natively configurable to run on custom networks, which the issue the PR is aimed to solve)

Reply:
@iccicci There is no issue preventing it from running. Since this is a draft PR, the feature is not yet available.